vhdbuilder/packer/sha1rsa_custom_data

```diff
@@ -0,0 +1,6 @@
+#cloud-config to reenable sha1 temporarily for packer
```

Comment:
this is due to issues with packer ssh failing? 

can we just use some more secure config?

Reply:
This workaround was added because Packer was only capable of using SHA-1 RSA due to some [longstanding limitations ](https://github.com/golang/go/issues/39885)in the underlying golang crypto lib. Because Mariner 2.0 is using a newer version of OpenSSH that has SHA-1 disabled by default, we use this customdata to temporarily reenable it for the lifetime of the Packer run.

For other clouds they have [the option ](https://github.com/hashicorp/packer-plugin-amazon/pull/179) of using ed25519 as their temporary keypair, but that's [not supported](https://docs.microsoft.com/en-us/troubleshoot/azure/virtual-machines/ed25519-ssh-keys) on Azure.

The good news is this looks like it may have been addressed in the [packer-plugin-azure v1.0.7](https://github.com/hashicorp/packer-plugin-azure/releases/tag/v1.0.7) release at the start of May. 

@Camelron can we try removing this customdata and testing a build on the latest version of the azure packer plugin?

Reply:
Testing now

Reply:
The version of Packer we are using doesn't pull in the latest packer-plugin-azure. The earliest version of packer that does so is v1.8.1 (https://github.com/hashicorp/packer/commit/9bf17ec95292f5729510cdce02828e58287f63fe). Added a todo comment to this workaround so we can remove it when we are able to.